src/main/java/net/kaczmarzyk/spring/data/jpa/domain/EqualDay.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2014-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.kaczmarzyk.spring.data.jpa.domain;
+
+import net.kaczmarzyk.spring.data.jpa.utils.Converter;
+import net.kaczmarzyk.spring.data.jpa.utils.QueryContext;
+
+import javax.persistence.criteria.*;
+import java.util.Objects;
+
+import static net.kaczmarzyk.spring.data.jpa.utils.DateTimeUtils.endOfDay;
+import static net.kaczmarzyk.spring.data.jpa.utils.DateTimeUtils.startOfDay;
+
+
+public class EqualDay<T> extends PathSpecification<T> {
```

Comment:
please add javadoc with a short description (can be the same or shorter than in readme

Reply:
(add author directive if you wish)